source/2015-04-13-1-year-of-cocoapods.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 1 Year of CocoaPods
+date: 2015-04-13 14:30 EDT
+tags:
+---
+
+A year ago today, I made my first serious attempt at contributing to CocoaPods. Little would I know that a late Saturday night (that quickly turned into Sunday morning after I returned from a party) spent hacking on some Ruby would forever change the course of history. Well, maybe not *history*, but certainly my own story, and that of CocoaPods.
+
+<!-- more -->
+
+I’ve been a CocoaPods user since the spring of 2013, my formative days as an iOS developer, before I even knew how to write a coherent program in Ruby. I couldn’t tell you who had written CocoaPods, whether it was sponsored by a corporation, or whether it would be around in a year, but from day 1 I was heavily indebted to it. I wouldn't hesitate to say that, without CocoaPods, I could not have made it as an iOS developer.
+
+The first time I was ever forced to consider the idea that CocoaPods was a concrete thing, a human production, was when I saw the announcement on the very blog of the Bug Bash. It was set for my last day of my very first college spring break, but I didn’t have anything better to do (I had been sidelined by two removed wisdom teeth earlier in the week), so I hopped on Metro North towards Manhattan, and Artsy HQ.
+
+Not knowing who I would meet, I took the elevator up to the 27th floor, and was promptly greeted by Orta. I sat down next to maybe five or six other people, and proceeded to triage issues. If memory serves, I had attended to the second-most issues that weekend, after Boris (hence his nickname, the Triagemaster General). I tried making one small fix to the specs, but it was really off-base, and I closed the pull request after a few minutes. Other than that, I had contributed no actual code to CocoaPods, but in trying to track down some of the issues, I learned a bit about CocoaPods worked internally. Even more importantly, however, I’d met Orta, along with chatting with Eloy, Fabio, and Kyle over IRC. It was a fun day, but still, it was no harbinger of the year to come. But it was a start.
```

Comment:
maybe link to the about page on Boris' name to make it make sense ( think you can directly jump to team with a fragment too )